crates/db_views/src/structs.rs

```diff
@@ -120,3 +122,8 @@ pub struct SiteView {
   pub local_site_rate_limit: LocalSiteRateLimit,
   pub counts: SiteAggregates,
 }
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct CustomEmojiView {
+  pub custom_emoji: CustomEmoji,
+  pub keywords: Vec<CustomEmojiKeyword>,
```

Comment:
Very bad, a view is a representation of a table, and it should not contain a list.

Reply:
A view is usually a combination of multiple tables, which this is. If you want to rename, do you have a suggestion?